elastic/kibana
# [Fleet] Fix repeated debug logs when bundled package directory does not exist

## Summary

Fixes https://github.com/elastic/kibana/issues/132621

Check for the bundled package dir once during setup, then squash repeated logs if the directory doesn't exist.

![image](https://user-images.githubusercontent.com/6766512/191990102-98e9f66c-f7e5-4482-bd40-db06cb6dde0e.png)

## To test

- Make sure `x-pack/plugins/fleet/target/bundled_packages` does not exist
- Run Fleet setup and verify a warning is logged
- Install packages and note that the no further directory missing log messages are printed
